tests/integration-tests/clusters_factory.py

```diff
@@ -88,6 +102,7 @@ def delete(self, delete_logs=False):
         """Delete this cluster."""
         if self.has_been_deleted:
             return
+        self.cfn_stack_arn  # Cache cfn stack arn before stack deletion.
```

Comment:
nit: I'd rather initialize self.__cfn_stack_arn in create() function

Reply:
Done